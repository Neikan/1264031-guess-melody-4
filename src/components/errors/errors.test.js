import React from "react";
import renderer from "react-test-renderer";
import Errors from "./errors.jsx";

describe(`Should Errors render correctly`, () => {
  test(`No errors`, () => {
    const tree = renderer
      .create(
          <Errors
            count={0}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  test(`With one error`, () => {
    const tree = renderer
      .create(
          <Errors
            count={1}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
