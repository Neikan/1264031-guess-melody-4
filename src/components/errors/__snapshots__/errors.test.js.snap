// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Errors render correctly No errors 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`Should Errors render correctly With one error 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;
